libs/email/components/content.tsx

```
@@ -2,33 +2,125 @@
         <tr>
-          <td className="px-10 py-0">
+          <td className="px-10 pb-10">
             <Card title={title}>
               <table className="w-full">
                 <tbody>
+                  <tr>
+                    <td className="text-center w-full">
+                      <pre className="text-sm text-[#e6e6e6] font-mono text-left text-wrap">
+                        You're registered for cuHacking 2025 at Carleton University! 🔥🔥🔥
+                        <br></br>
+                        We're excited to have you join us for a weekend of hacking, learning, and fun!!!
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)